infinite_tracing/lib/infinite_tracing/channel.rb

```diff
@@ -35,6 +58,15 @@ def settings
           'grpc.enable_deadline_checking' => 0
         }
       end
+
+      def validate_compression_level
+        NewRelic::Agent.logger.debug("Infinite Tracer compression level set to #{compression_level}")
+
+        return if COMPRESSION_LEVELS.include?(compression_level)
+
+        raise "Invalid compression level '#{compression_level}' specified! Must be one of " \
```

Comment:
Where does this raise end up getting caught? Would this prevent infinite tracing from starting completely? I don't think we would want to break infinite tracing if someone did something incorrect for their config settings. 
It might be good if, instead of raising here, we logged a warning and then set the compression level to be whatever the default is.

Reply:
Good call - we just log instead of raising. Fixed with 873ed0a48aabb6c4137d69c2bc5a20457733553c